ui/views.py

```diff
@@ -287,7 +287,7 @@ def serve_static_assets(request, path):
         raise PermissionDenied()
     filename = os.path.basename(file_path)
     response = StreamingHttpResponse(
-        FileWrapper(open(file_path), 'rb'),
+        FileWrapper(open(file_path, 'rb')),
```

Comment:
It just returned an empty response. I think there was an exception since the `'rb'` was passed where a block size should have been, and somehow the error was just swallowed up.